tests/pgautofailover_utils.py

```diff
@@ -1093,6 +1093,9 @@ def run(self, env={}):
         self.pg_autoctl = PGAutoCtl(self)
         self.pg_autoctl.run(level='-v')
 
+        # when on the monitor we always want Postgres to be running to continue
+        self.wait_until_pg_is_running()
```

Comment:
Before this patch we would run Postgres at `pg_autoctl create monitor` time, even when not using `--run`, and Postgres would keep running in the background. Now, if our own process does not run, neither Postgres runs. Rather than editing all the call sites to wait until Postgres has started when doing `pg_autoctl run` on the monitor, it seems easier to just do it here, always.